.vscode/settings.json

```diff
@@ -31,5 +31,8 @@
         }
       }
     }
-  ]
+  ],
+
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true
```

Comment:
Format on save is a bit opinionated, idk...

Reply:
My idea was this would replace the Prettier action but I don't think most poorly formatted PR authors are using VSCode anyway. I can remove it.

Reply:
No, I'll undo that.